woocommerce/woocommerce-gutenberg-products-block
# Use backslashes to fix webpack build on windows

This PR will modify the webpack config to test for Windows directory separators too in case someone is developing on a Windows machine.

In #6797 I reported that certain Sass files were not loading and it is because the test did not check for Windows directory separators (which use a backslash  instead of a forward slash).



Fixes #6797
Might fix #7440 (investigation required)

### Testing

#### Automated Tests

#### Manual testing
1. Get a Windows 10 or 11 machine
2. Run `npm run build` and ensure no errors show up.
3. Smoke test the blocks and ensure styles are working properly.
4. Do the same on Mac or Linux and check step block and styles are working correctly.

#### User Facing Testing



None required.


### WooCommerce Visibility




### Changelog

> Developers: Fixed an issue where the project would not build on Windows machines.
